includes/Flows/Upgrades/1.0.0.php

```diff
@@ -12,8 +12,8 @@
 	if ( ! empty( $flow_data['data']['palette']['slug'] ) ) {
 		$flow_data['data']['colorStyle'] = $flow_data['data']['palette']['slug'];
 	}
-	if ( ! empty( $flow_data['data']['typography']['slug'] ) ) {
-		$flow_data['data']['fontStyle'] = $flow_data['data']['typography']['slug'];
+	if ( ! empty( $flow_data['data']['fonts']['slug'] ) ) {
+		$flow_data['data']['fontStyle'] = $flow_data['data']['fonts']['slug'];
```

Comment:
Let's leave `typography` as it is in this file; this is meant for upgrades to users with older versions of flow data.